proposals/0357-alpenglow_validator_admission_ticket.md

```diff
@@ -0,0 +1,219 @@
+described in [SIMD 326](https://github.com/solana-foundation/solana-improvement-documents/pull/326)
+will be implemented. Specifically, how it affects validator operation
+procedures.
+
+The validator admission ticket is a mechanism translating the current cost of
+voting into a similar economic equilibrium for Alpenglow. By charging every
+voting validator 1.6 SOL per epoch, it replaces the current voting fee at ~2
+SOL per epoch, and reduces the likelihood there are too many voting validators
+immediately after Alpenglow launches.
+
+The general VAT concept has already been accepted with the governance vote on
+SIMD 326.
+
+Everything specified below are protocol-level changes, they need to be
+implemented by all Solana clients. The client specific data structure
+changes are omitted.
+
+## Dependencies
+
+The Validator Admission Ticket is specified in SIMD 326 (Alpenglow).
+
+The compressed BLS public key in Vote Account is specified in [SIMD 185 (Vote
+Account v4)](https://github.com/solana-foundation/solana-improvement-documents/pull/185)
+
+## Motivation
+
+Adding more staked validators to a blockchain does come with costs. At the
+very least, the votes and corresponding rewards need to be processed by
+every voting validator. Therefore, every additional staked validator increases
+the number of messages for every validator.
+
+Right now every voting validator pays voting fee on any vote transaction
+included in a block. The voting fee adds up to ~2 SOL per epoch if the
+validator votes most of the time. It is a burden, yet at the same time it's an
+economic barrier to having too many voting validators.
+
+VAT is only a temporary solution to maintain the current economic equilibrium.
+This proposal intentionally strives to keep voting validator protocol costs
+similar to pre-Alpenglow consensus.
+
+## New Terminology
+
+- **Validator Admission Ticket(VAT)**: The 1.6 SOL charged once per epoch to
+every validator eligible to participate in the next epoch
+
+## Different Vote Related Accounts
+
+Before Alpenglow, we have the following accounts related to voting (the
+identity account and vote authority account can be the same):
+
+- Vote account for saving all the vote states and receiving commission
+
+- Identity account for receiving block rewards
+
+- Vote authority account for signing vote transactions
+
+After Alpenglow, we will still have roughly the same accounts serving the same
+purpose:
+
+- Vote account: This account must contain the correct BLS public key
+corresponding to vote authority keypair. It continues to keep all the vote
+credits and all validator identity/authority/commission information updates
+happen here, but it doesn’t contain vote information any more.
+
+- Identity account: This is the account for the 1.6 SOL VAT; it continues
+receiving block rewards and commissions.
+
+- Vote authority account: For signing BLS messages
+
+## Detailed Design
+
+### What is considered a valid vote account
```

Comment:
this section seems wholly duplicative of the next. i don't think we need them both